From TeX Live svn rev 23421.
Remove unnecessary headers. Fixes build with kpathsea from TeX Live 2012.


Index: texlive-20110705-source/texk/web2c/luatexdir/luatex.c
===================================================================
--- texlive-20110705-source.orig/texk/web2c/luatexdir/luatex.c
+++ texlive-20110705-source/texk/web2c/luatexdir/luatex.c
@@ -32,7 +32,6 @@ const char *engine_name = "luatex";
 #include <kpathsea/readable.h>
 #include <kpathsea/variable.h>
 #include <kpathsea/absolute.h>
-#include <kpathsea/recorder.h>
 #ifdef WIN32
 #include <kpathsea/concatn.h>
 #endif
Index: texlive-20110705-source/texk/web2c/lib/texmfmp.c
===================================================================
--- texlive-20110705-source.orig/texk/web2c/lib/texmfmp.c
+++ texlive-20110705-source/texk/web2c/lib/texmfmp.c
@@ -17,7 +17,6 @@
 #include <kpathsea/readable.h>
 #include <kpathsea/variable.h>
 #include <kpathsea/absolute.h>
-#include <kpathsea/recorder.h>
 #ifdef WIN32
 #include <kpathsea/concatn.h>
 #endif
